.expeditor/verify.pipeline.yml

```diff
@@ -19,15 +19,15 @@ steps:
 
   - label: run-tests-ruby-3.1
     command:
-      - /workdir/.expeditor/buildkite/verify.sh
+      - K=16 RAKE_TASK=test:parallel /workdir/.expeditor/buildkite/verify.sh
```

Comment:
Sathish we don't want this for linux pieplines only needed for Windows

Reply:
Those are not slow, as per my understanding and do not use threads currently. Also do not face timeout issues

Reply:
No @Vasu1105 , they were slow see other verify pipelines https://buildkite.com/chef-oss/inspec-inspec-inspec-5-verify/builds/1004 , https://buildkite.com/chef-oss/inspec-inspec-inspec-5-verify/builds/1004 

Previous verify pipelines take ~40 mins to complete this PR reduces it to ~4 mins

And Inspec test does support K value to control the threads https://github.com/inspec/inspec/blob/main/Rakefile#L74 and https://github.com/inspec/inspec/blob/main/Rakefile#L133